.github/workflows/pypi-express-relay-utils.yml

```diff
@@ -14,8 +14,10 @@ jobs:
       - name: Install dependencies
         run: |
           python3 -m pip install --upgrade poetry
-          poetry -C express_relay/sdk/python/express_relay build
+          cd express_relay/sdk/python/express_relay
+          poetry build
       - name: Build and publish
         run: |
-          poetry -C express_relay/sdk/python/express_relay build
-          poetry -C express_relay/sdk/python/express_relay publish --username __token__ --password ${{ secrets.PYPI_TOKEN }}
+          cd express_relay/sdk/python/express_relay
```

Comment:
I think it's better to use working-directory like here https://github.com/pyth-network/pyth-crosschain/blob/main/.github/workflows/publish-pyth-sdk-cw.yml#L18